main/guides/js-programming/notifiers.md

```diff
@@ -255,15 +255,15 @@ non-final values due to `NotifierKit()`'s lossy nature.
 
 ## Distributed Operation
 
-Either make `NotifierKit()` or `makeSubscriberKit()` can be used in a multicast
+Either make `NotifierKit()` or `makeSubscriptionKit()` can be used in a multicast
 manner with good distributed systems properties, where there is only one
 producing site but any number of consuming sites. The producer is not vulnerable
 to the consumers; they cannot cause the kit to malfunction or prevent the code
 producing values from making progress. The consumers cannot cause each other to
 hang or miss values. 
 
-For distributed operation, all the iteration values---non-final values,
-successful completion value, failure reason---must be `Passable`; values that
+For distributed operation, all the iteration values—non-final values,
```

Comment:
I think there's some way to express an m-dash. If not, triple dash is still a better approximation than single dash.

Reply:
This is an em dash, which is hard to see in the GitHub source view but easier in the rendered Markdown: https://github.com/Agoric/documentation/blob/95d1f32a02d75ef5defffdb49d7aedcd6c391d8e/main/guides/js-programming/notifiers.md#distributed-operation

Alternatively, it could be made even more clear in the source at the loss of readability:
```suggestion
For distributed operation, all the iteration values&mdash;non-final values,
```

Reply:
I prefer the `&mdash;` in the source. In general, I prefer sources in ascii, so I'm more likely to guess right when interpreting what I'm looking at when reading source. When the thing in the source is clearly non-ascii from how it appears in the source view, that can be ok. I really want to avoid source that looks like ascii but isn't. In fact, it would be nice to have a lint-rule to alert us to those cases.